model/survey-response-logger.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+/**
+ * @module model/survey-response-logger
+ */
+
+const Sequelize = require('sequelize');
+
+/**
+ * Registers model with Sequelize
+ * @param {Sequelize} sequelize - database instance
+ * @returns {Null} nothing
+ */
+function register (sequelize) {
+    /**
+     * Each SurveyResponseLogger will be associated with a completed SurveyInstance.
+     * Each record represents the timestamps associated with when the question was answered.
```

Comment:
This stores data only for questions that were answered?
Should this be part of the `question_result` table?